model/log/empty_iterator.go

```diff
@@ -0,0 +1,19 @@
+package log
+
+var empty = &emptyIterator{}
+
+type emptyIterator struct{}
+
+// EmptyIterator returns a convenience log iterator with no data that returns
+// false immediately.
+func EmptyIterator() *emptyIterator { return empty }
```

Comment:
It seems like this could just return a `&emptyIterator{}` rather than store the empty iterator once in a package-level field.